faster_whisper/transcribe.py

```diff
@@ -150,6 +151,7 @@ def transcribe(
         self,
         audio: Union[str, BinaryIO, np.ndarray],
         language: Optional[str] = None,
+        return_all_language_probs: bool = False,
```

Comment:
I'm OK to always return the language probabilities when they are available. So I suggest to simply remove this argument.

Reply:
Sure, thanks! To confirm, we'll still keep the top language name and prob for backward compatibility right?

Reply:
Yes, keep the existing properties and add `all_language_probs` (which can still be `None` if the language detection did not run).